### ISSUE TITLE
Image with alt text in an anchor should pass.

### ISSUE THREAD

**Issue by DavidMacDonald** [dequelabs/axe-core#597]
AXE is reporting a failure for this.

```<a href="http://www.bayer.ca/en/home/" target="_blank"><img class="visible-xs cs_bayer_logo_mo" src="/static/media/images/bayer_logo.jpg" alt="Bayer logo"></a>```

I think it needs to pass links that have only images with alt text.

**Comment by marcysutton** [comment]
@DavidMacDonald which version of aXe? I don't get that result, it passes for me.

**Comment by DavidMacDonald** [comment]
![image](https://user-images.githubusercontent.com/2365411/32291919-d45f45ce-bf14-11e7-96a6-cbbbf39a008c.png)

**Comment by marcysutton** [comment]
@DavidMacDonald I'm in the exact same version and I don't get that result. Is there something else on the page that's causing interference? Can you provide more details about the node, rule id, etc.?

**Comment by marcysutton** [comment]
Check to see if there's another image on the page somewhere that's triggering it, by using the inspect or highlight functions.

**Comment by DavidMacDonald** [comment]
![image](https://user-images.githubusercontent.com/2365411/32297330-3d44f5ce-bf25-11e7-98de-ac98e4a6dc3a.png)

**Comment by WilcoFiers** [comment]
That's so crazy! Can you share the page you found this on? Exploring this site myself didn't get me anything.

**Comment by DavidMacDonald** [comment]
https://www.xarelto.ca/

**Comment by DavidMacDonald** [comment]
hmmm.... the same page on my laptop doesn't have that error.... the plot thickens.
Could be a weird caching problem, but its weird AXE pulled the anchor tag and correctly showed the  code, and called it an error...

**Comment by marcysutton** [comment]
Ok, now that we've got the whole page I do get the error. The problem is that image has `display: none` on it, so the link in fact doesn't have text in it. The HTML alone is fine, but the CSS makes it inaccessible. aXe is doing the correct thing here, so I'm closing the issue.

**Comment by DavidMacDonald** [comment]
why does the screen reader see the link text I wonder? https://github.com/dequelabs/axe-core/issues/597#issuecomment-341204801

**Comment by marcysutton** [comment]
@DavidMacDonald there's more than one link with that image in it. Judging by the CSS class, the one that got flagged is probably for a small/mobile viewport.
